uint16[] web_motor_msg
string direction
string duration
